libraries/cyclestreets-view/src/main/java/net/cyclestreets/routing/Journey.kt

```diff
@@ -200,6 +208,29 @@ class Journey private constructor(wp: Waypoints? = null) {
                 journey.elevation.add(sdo.segmentProfile)
             }
         }
+        // For circular routes.
+        private fun populatePois(jdo: JourneyDomainObject, context: Context) {
```

Comment:
what stops this running for non-circular routes?

Reply:
It will run, but jdo.pois will be empty, so nothing will happen.
Shall I put a condition around the call, or leave it as is in case there is ever a feature introduced to add POIs to a linear route? ;-)

Reply:
fine as-is